lib/maze/bitbar_utils.rb

```diff
@@ -47,6 +50,35 @@ def upload_app(api_key, app)
         end
         app_uuid
       end
+
+      # Starts the BitBar local tunnel
+      # @param bb_local [String] path to the SBSecureTunnel binary
+      # @param username [String] Username to start the tunnel with
+      # @param access_key [String] BitBar access key
+      def start_local_tunnel(bb_local, username, access_key)
+        $logger.info 'Starting BitBar SBSecureTunnel local tunnel'
+        command = "#{bb_local} --username #{username} --authkey #{access_key}" \
+                    "--ready #{BB_READY_FILE} --kill #{BB_KILL_FILE}"
+
+        @tunnel_shell = Maze::InteractiveCLI.new
+        @tunnel_shell.run_command(command)
+        success = Maze::Wait.new(timeout: 30).until do
+          File.exist?(BB_READY_FILE)
+        end
+        unless success
+          $logger.info "Failed: #{@tunnel_shell.stdout_lines}"
+        end
+      end
+
+      # Stops the local tunnel
+      def stop_local_tunnel
+        FileUtils.touch(BB_KILL_FILE)
+        Maze::Wait.new(timeout: 30).until do
+          !@tunnel_shell.running?
+        end
+        File.delete(BB_READY_FILE) if File.exist?(BB_READY_FILE)
```

Comment:
Is it worth deleting these if they exist before starting the tunnel as well?  I know I can be very Command-C happy, potentially leaving files lying around.